src/components/CompareResults/overTimeLoader.ts

```diff
@@ -46,9 +66,6 @@ function checkValues({
     );
   }
 
-  if (interval === null) {
-    interval = 86400; // default to 1 day
-  }
   const intervalValue = +interval as TimeRange['value'];
```

Comment:
I just noticed that the return values below do not return these values, but instead they return hardcoded values :) This should be fixed (in this PR or in another PR as you wish)

Reply:
That's silly. I don't know why I did that. I must have been testing something and forgot to revert it. I'll fix it in this PR. Thanks for catching that!